src/stormpath.social-login.linkedin.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+angular.module('stormpath.linkedinLogin', [])
+.provider('$linkedinLogin', function() {
+  function loadLinkedinSDK(clientId, onload) {
+    var id = 'linkedin-jssdk';
+    var firstJs = document.getElementsByTagName('script')[0];
+    var js;
+
+    if (document.getElementById(id)) {
+      return;
+    }
+
+    js = document.createElement('script');
+    js.id = id;
+    js.src = '//platform.linkedin.com/in.js';
+    js.onload = onload;
+    js.innerHTML = 'api_key: ' + clientId + '\n' +
+                   'onLoad: ' + onload.name + '\n' +
```

Comment:
This does not work for anonymous methods. Would be nicer to wrap the callback and generate a function that you attach on the window scope, then use the name of that function.